authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import serializers
+from .models import Article
+from ..authentication.models import User
+from taggit_serializer.serializers import (TagListSerializerField,
+                                           TaggitSerializer)
+
+class CreateArticleAPIViewSerializer(TaggitSerializer,serializers.ModelSerializer):
+    tags = TagListSerializerField()
+    author = serializers.SerializerMethodField()
+
+    def get_author(self,obj):
+        user = {
+            "username":obj.author.username,
+            "email":obj.author.email
+        }
+        return user
+
+
+    class Meta:
+        model = Article
+
+        fields = ['title','description', 'body', 'author', 
+                    'created_at', 'updated_at', 'tags', 'slug', 'published', 'image']
+
+    def validate_title(self, value):
```

Comment:
The title definition in the model already has the `max_length` attribute, you could as well as add the `min-length` if you wanted to validate that constraint so that you don't have to write the entire validation logic.

Reply:
I really don't think we need to limit people here,  let me give an example when you are to post on medium.com, even if its one letter you cannot be limited to have that, but for the sake of maximum length, the title doesn't really have to be so long. It can't be like a body